expyfun/_input_controllers.py

```diff
@@ -489,3 +532,23 @@ def _retrieve_events(self, live_keys):
             if live_keys is None or key[0] in live_keys:
                 targets.append(key)
         return targets
+
+    def _retrieve_keyboard_releases(self, live_keys):
+        # add escape keys
+        if live_keys is not None:
+            live_keys = [str(x) for x in live_keys]  # accept ints
+            live_keys.extend(self.force_quit_keys)
+        # pump for events
+        self._dev.poll_for_response()
+        while self._dev.response_queue_size() > 0:
+            key = self._dev.get_next_response()
+            if not key['pressed']:
```

Comment:
This is a way to implement it for `CedrusBox`: if `key['pressed']` is `False`, it's a release event. 

I don't know how to test this though. Do we have one of these at I-LABS?